compose-destinations/build.gradle.kts

```diff
@@ -2,6 +2,7 @@ plugins {
     id("com.android.library")
     kotlin("android")
     id("com.vanniktech.maven.publish")
+    kotlin("plugin.serialization")
```

Comment:
Hmmm I'd like to avoid this if possible. I don't known yet exactly how.. I'm guessing we can also generate the serializers like we do for navigation on the user's module and have them point to them by passing their instance or something like this.

To be honest, if this is the only way, I'd rather not do it. I don't want to force this dependency on all users of the library. Maybe we could make this a separate dependency, but that also comes with additional overhead for all maintainers...